src/ui/App.tsx

```diff
@@ -27,6 +27,105 @@ enum LoadState {
   loaded,
 }
 
+// TODO: Add setaside
+enum PriorityVisibility {
+  visible,
+  hidden,
+}
+
+enum BuiltInAction {
+  archive,
+  moveToPriority,
+}
+
+// If the action value is a string, it's the name of the priority to move to.
+type Action = BuiltInAction | string;
```

Comment:
We'll be representing priorities as gmail labels. Should we wrap this in something making it easier to manager mapping from an action to the associated gmail label?

Reply:
I was thinking the string would be the gmail label name. Is there other metadata we'd also want to track?

Reply:
I was thinking we might want to directly include a "Label" object.
https://github.com/ojanvafai/tskod/blob/6f706f62b328caa920ec8c17e4b3a8c15ecf165e/src/Labels.tsx#L19

Not positive that makes sense though. Managing lazy instantiation might be a pain.